cmd/src/doc.go

```diff
@@ -97,7 +99,9 @@ Examples:
 					}
 					defer file.Close()
 
-					file.WriteString(content)
+					if _, err := file.WriteString(content); err != nil {
+						return err
```

Comment:
This would break out of the loop, which is _maybe_ what we want, but just double checking the assumption :slightly_smiling_face:

Reply:
I think that's fine, yeah.